test_expect_failure/missing_direct_deps/scala_proto_deps/customized_scala_proto.bzl

```diff
@@ -0,0 +1,34 @@
+load("//scala_proto:scala_proto.bzl", "make_scala_proto_aspect", "make_scala_proto_library")
+load("@io_bazel_rules_scala//scala:advanced_usage/providers.bzl", "ScalaRulePhase")
+
+def _phase_custom_stamping_convention(ctx, p):
+    rule_label = str(p.target.label)
+    toolchain = ctx.toolchains["@io_bazel_rules_scala//scala_proto:toolchain_type"]
+
+    if toolchain.stamp_by_convention:
+        return rule_label + "_scala"
```

Comment:
Maybe you could use suffix that stands out better. It was hard to spot difference between `some_scala_proto` and `some_proto_scala`